src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/WSHttpTransportSecurityElement.cs

```diff
@@ -43,5 +43,17 @@ public string Realm
                 base[ConfigurationStrings.Realm] = value;
             }
         }
+
+        internal void ApplyConfiguration(HttpTransportSecurity security)
+        {
+            if (security == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(security));
+            }
+
+            security.ClientCredentialType = ClientCredentialType;
+            security.Realm = Realm;
+           // security.ExtendedProtectionPolicy = ChannelBindingUtility.BuildPolicy(this.ExtendedProtectionPolicy);
```

Comment:
You forgot to remove the commented out property for ProxyCredentialType above when you removed the commented setting code here.